data/src/test/java/org/apache/iceberg/data/TestLocalScan.java

```diff
@@ -521,30 +508,7 @@ public void testAsOfTimeOlderThanFirstSnapshot() {
             "Cannot find a snapshot older than " + DateTimeUtil.formatTimestampMillis(timestamp));
   }
 
-  private DataFile writeFile(String location, String filename, List<Record> records)
```

Comment:
can you please move these methods back as that doesn't seem necessary to move them around